plugin/java/src/test/java/com/buschmais/jqassistant/plugin/java/test/scanner/SuppressIT.java

```diff
@@ -69,4 +73,43 @@ private void verifySuppress(String constraintId, String conceptId, String column
         store.commitTransaction();
     }
 
+    @Test
+    void suppressUntilWithMonthsLimit() throws RuleException {
```

Comment:
The annotated elements should be verified (see comment above for the result structure of the constraints)